site/index.ejs

```diff
@@ -36,6 +36,74 @@
         }
     </script>
     <title><% if (typeof title !== 'undefined') {%><%= title %><% } %></title>
+
+    <script>
+      var colourOnButtonText = 'Make my day accessible';
```

Comment:
One more thing @matt-paul - could we add a comment here explaining what this is and how it's supposed to work with the other (button) PR and that there is a similar script in blog? Just conscious of people maybe doing this next year and it not necessarily being clear on what is needs to be done.

Reply:
👍